### PR TITLE
Remove all Docker.io References from images for Secure Supply Chain

### PR DESCRIPTION
**Pull request by CooperLink**
Remove a list of old and deprecated images from this repository.  Yesterday, the Microsoft Secure Supply Chain utility was blocking builds with a failure because of docker.io references in this repository.  This PR removes all unnecessary images in this repo that reference Docker.io. 

Java8 images in v2 - Remove Alpine and Stretch non-appservice images

Python images in v2 - Remove all Python v2 images

Kudulite - Remove deprecated Kudulite images. Builds haven't run since I started at Microsoft and the code has been migrated. 

> https://azure-functions.visualstudio.com/azure-functions-docker/_build?definitionId=13&_a=summary

> https://msazure.visualstudio.com/Antares/_git/AAPT-Antares-KuduLite

Arm32v7 - Remove Arm32v7 tests.  **This may break the CI testing for Dotnet v3 Arm32v7 images** but, we do not take any dependency on them.  I am open to discussion on this.  I can try to work around the docker io problems if  needed

Python v3/v4  - Remove all images temporarily to try and pass the SSC test. I will be adding these back when Prashant gives me the other image definition files. 

Releasing new Host versions.
[V3 Release](PASTE_V3_LINK_HERE)
[V2 Release](PASTE_V2_LINK_HERE)

---

### PR information

- [x] The title of the PR is clear and informative.
- [ x There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [x] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

**Review by pragnagopa — COMMENTED** [review]
ACK on removing 2.0 images and KuduLite Images.

Unsure about grpc/strecth/arm32v7 - is this used in V3?

**Review by pragnagopa — DISMISSED** [review]
Please go ahead and remove 2.0 folder - not needed

File: host/4/bullseye/amd64/python/python37/python37.Dockerfile [hunk 1]
```diff
@@ -34,9 +34,23 @@ RUN apt-get update && \
     rm -f /$EXTENSION_BUNDLE_FILENAME_V3 &&\
     find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
 
-FROM python:3.7-slim-bullseye
+FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
```
Comment (CooperLink) [inline host/4/bullseye/amd64/python/python37/python37.Dockerfile]: We can optimize these Python images once they are updated.  Optimization will allow us to share layers more effeciently in the platform.

**Comment by CooperLink** [comment]
> Please go ahead and remove 2.0 folder - not needed

Removed

**Comment by CooperLink** [comment]
Got Ack from @ahmelsayed that GRPC image can be removed from this repository.

File: host/4/bullseye/amd64/python/python37/python37.Dockerfile (hunk 1 above)
Comment (pragnagopa) [reply host/4/bullseye/amd64/python/python37/python37.Dockerfile]: cc @paulbatum - python images are moving to the common base layer as other images !

**Review by pragnagopa — DISMISSED** [review]
@CooperLink  - please create a v2.x branch and push the 2.0 folder to that branch - just in case if we ever need them

**Comment by CooperLink** [comment]
I persisted the v2 images to a branch called v2.0-images.  I removed them from this branch.  

Discussed with Varad and Prashant.  Removing Python changes here. They will get those changes through before the end of the month.

**Review by pragnagopa — APPROVED** [review]
Please

**Pull request merged**